builder.config.json

```diff
@@ -72,12 +72,12 @@
       {
         "id": "22",
         "label": "English",
-        "iso_639_3": "EN"
+        "iso_639_3": "eng"
```

Comment:
Can we also update these pls:
- https://github.com/FLOIP/flow-builder/blob/VMO-3951-hook-import-into-basic-validations/src/store/builder/2019-10-10-shortcut-flow.json#L18
- https://github.com/FLOIP/flow-builder/blob/VMO-3951-hook-import-into-basic-validations/stories/Flow.stories.ts#L57

Reply:
Thanks for changing them